www/redirects.yaml

```diff
@@ -150,3 +150,5 @@
   toPath: /contributing/docs-and-blog-components/
 - fromPath: /docs/pixabay-source-plugin-tutorial
   toPath: /docs/source-plugin-tutorial
+- fromPath: /docs/page-build-optimizations-for-incremental-data-changes
```

Comment:
Thanks for covering all the bases 🙂 

Could you also replace the old listing for the doc in the [`doc-link.yml`](https://github.com/gatsbyjs/gatsby/blob/c72a9748cd7e2cd3c8054c6b708c41d202e1d939/www/src/data/sidebars/doc-links.yaml#L528-L529)?
 https://github.com/gatsbyjs/gatsby/blob/c72a9748cd7e2cd3c8054c6b708c41d202e1d939/www/src/data/sidebars/doc-links.yaml#L528-L529

That's where docs are explicitly added to the sidebar, so it'll show up with the correct name.

Reply:
Thanks!